src/pytest/test_debug_commands.py

```diff
@@ -175,28 +175,28 @@ def testGitSha(self):
         self.env.expect('FT.DEBUG', 'git_sha', 'foo').notRaiseError()
 
 
-class TestDebugCommandsLogAssert(object):
+#class TestDebugCommandsLogAssert(object):
     # until added to RLtest
-    def skipOnRedisVersion(self, ver):
```

Comment:
@ashtul did you checked it? any idea how it pass on redis 5?

Reply:
@MeirShpilraien Not sure and removed